pkg/java/src/main/java/dev/openfga/language/validation/ValidationErrorsBuilder.java

```diff
@@ -24,7 +24,7 @@ private ErrorProperties buildErrorProperties(String message, int lineIndex, Stri
 
         var rawLine = lines[lineIndex];
         var regex = Pattern.compile("\\b" + symbol + "\\b");
-        var wordIdx = 0;
```

Comment:
This aligns with the change I made to JS to ensure we return a 1 based column number always